share/base.css.in

```diff
@@ -79,7 +80,7 @@
   position:relative;
   background-color:white;
   overflow: hidden;
-  margin:0; 
+  margin:0 auto; 
```

Comment:
On Safari the `CSS_PAGE_FRAME_CN` class with `margin:0;` results in the content shifted to left instead of staying at center, adding `auto` fixed it.